Dockerfile

```diff
@@ -1,6 +1,4 @@
 FROM tsutomu7/alpine-python-pandas
-ADD . /autoapi
```

Comment:
Moving this to the *end* of the file is crucial; when it's at the beginning, it means that the slightest change to any file in the source directory will trigger a full rebuild of the whole image, which takes a long time and also means a full re-download for anyone who's downloading the image from docker hub.